guides/keras_cv/semantic_segmentation_deeplab_v3_plus.py

```diff
@@ -78,6 +76,51 @@
 
 from keras_cv.datasets.pascal_voc.segmentation import load
 
+"""
+## Perform semantic segmentation with a pretrained DeepLabV3Plus model
+
+The highest level API in the KerasCV semantic segmentation API is the keras_cv.models
+API. This API includes fully pretrained semantic segmentation models, such as
+keras_cv.models.DeepLabV3Plus.
+
+Let's get started by constructing a DeepLabV3Plus pretrained on the pascalvoc dataset.
+"""
+
+model = keras_cv.models.DeepLabV3Plus.from_preset(
+    "deeplab_v3_plus_resnet50_pascalvoc",
+    num_classes=21,
+    input_shape=[512, 512, 3],
+)
+
+"""
+Let us visualize the results of this pretrained model
+"""
+
+filepath = tf.keras.utils.get_file(origin="https://i.imgur.com/A8eQsll.jpeg")
```

Comment:
Awesome -- thank you!

This LGTM, but I do think we should use an image that doesn't have a logo in it (I see this one has a little watermark in the bottom-right)

Reply:
I updated it to the one from object detection guide https://i.imgur.com/gCNcJJI.jpg